web/source/kmwnative.js

```diff
@@ -96,19 +96,17 @@
   */    
   keymanweb.keyboardUnavailable = function(Ln)
   {
-    return window.setTimeout(function() 
-      {
+    return window.setTimeout(function() {
         util.wait(false);
         var Ps=keymanweb._KeyboardStubs[Ln],kbdName=Ps['KN'],lgName=Ps['KL'];
         kbdName=kbdName.replace(/\s*keyboard\s*/i,'');
-        util.alert('Sorry, the '+kbdName+' keyboard for '+lgName+' is not currently available!',keymanweb.setDfltKeyboard);
+        util.alert('Sorry, the '+kbdName+' keyboard for '+lgName+' is not currently available!', keymanweb.setActiveKeyboard(''));
```

Comment:
This needs to be 
````
util.alert('Sorry, the '+kbdName+' keyboard for '+lgName+' is not currently available!',
  function() {
    keymanweb.setActiveKeyboard('');
  }
);
````

Otherwise, you are passing the result of `setActiveKeyboard` to `util.alert` rather than passing the function itself.